train.py

```diff
@@ -69,9 +69,16 @@
     train_loader = get_loader(opts, real=True, no_check=args.no_check)
     print("Creating display images...", end="", flush=True)
     val_iter = iter(val_loader)
+    
     test_display_images = [
         Dict(val_iter.next()) for i in range(opts.comet.display_size)
     ]
+    
+    if opts.train.save_im:   
+        train_iter = iter(train_loader)
+        train_display_images = [
+            Dict(val_iter.next()) for i in range(opts.comet.display_size)
```

Comment:
1 thing that bugs me with this method is why even ask for that many batches if only the first image of the batch is going to be used in test_networks anyway?